modules/test/playwright/tests/journal-web/journalArticle.spec.ts

```diff
@@ -637,6 +637,60 @@ baseTest(
 		await waitForAlert(page);
 	}
 );
+baseTest(
+	'It ensures that translate side by side shows the duplicate fields',
+	{
+		tag: '@LPS-142169',
+	},
+	async ({apiHelpers, journalEditArticlePage, journalPage, page, site}) => {
+		const localizableFieldName = 'Text5678';
+		const structureName = 'Structure 1';
+
+		const dataDefinition = getDataStructureDefinition({
+			defaultLanguageId: 'en_US',
+			fields: [{name: localizableFieldName, repeatable: true}],
+			name: structureName,
+		});
+
+		await apiHelpers.dataEngine.createStructure(site.id, dataDefinition);
+
+		await journalEditArticlePage.goto({
+			siteUrl: site.friendlyUrlPath,
+			structureName,
+		});
+		const title = getRandomString();
+
+		await journalEditArticlePage.titleInput.fill(title);
+
+		const field = page.locator(
+			'input[id^="_com_liferay_journal_web_portlet_JournalPortlet_ddm$$Text"]'
+		);
+
+		await fillAndClickOutside(page, field, 'Text Field');
+
+		await journalEditArticlePage.duplicateButton.click();
+
+		await page
+			.locator(
+				'input[id^="_com_liferay_journal_web_portlet_JournalPortlet_ddm$$Text"]'
+			)
+			.nth(1)
+			.fill('Duplicated Text Field');
+
+		await journalEditArticlePage.publishButton.click();
```

Comment:
I would move all this above steps (from L657) to a function in JournalEditArticlePage, something like `createArticleWithDuplicateField` 🤔 . So the test is more clear.